src/javascript/app/common/chart_settings.js

```diff
@@ -41,9 +41,7 @@ const ChartSettings = (() => {
         txt_subtitle = (params.is_chart_delayed ? labels.delay : '') +
             (params.is_forward_starting ? labels.purchase_time : '') +
             (params.is_sold_before_start ? '' : start_time) +
-            (history ? ((params.is_sold_before_start || params.is_tick_trade) ? '' : labels.entry_spot) : '') +
```

Comment:
this also removes it from tick trades, we only want to know why it's showing for candles and remove it from there